### PR TITLE
FIx unit tests in master

### PR DESCRIPTION
**Pull request by Siedlerchr**
FIx some tests related to Journal Abbreviation and some other minor issues

- [ ] Change in CHANGELOG.md described (if applicable)
- [x] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java [hunk 1]
```diff
@@ -125,4 +126,9 @@ public void addCustomAbbreviations(Collection<Abbreviation> abbreviationsToAdd)
     public Set<String> getFullNames() {
         return fullToAbbreviation.keySet();
     }
+
+    public List<Abbreviation> getBuiltin() {
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java:130]: I think this is a wrong semantics here. The built-in refers to the built-in journal list, but `fullToAbbreviation` is read from the file passed as the constructor argument.

File: src/main/java/org/jabref/logic/journals/JournalAbbreviationLoader.java [hunk 2]
```diff
@@ -35,13 +28,6 @@
         return parser.getAbbreviations();
     }
 
-    private static List<Abbreviation> readJournalListFromFile(Path file, Charset encoding) throws IOException {
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/logic/journals/JournalAbbreviationLoader.java:38]: With this method removed, how are custom journal abbreviation lists (csv files) loaded?

File: src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelMixedAbbreviationsTest.java [hunk 3]
```diff
@@ -45,6 +46,7 @@
     private Path testFile4Entries;
     private Path testFile5EntriesWithDuplicate;
     private JournalAbbreviationPreferences abbreviationPreferences;
+    private final JournalAbbreviationRepository repo = JournalAbbreviationLoader.loadBuiltInRepository();
```
Comment (tobiasdiez) [inline src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelMixedAbbreviationsTest.java:49]: maybe rename to builtinRepo?

File: src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelNoShortestUniqueAbbreviationsTest.java [hunk 4]
```diff
@@ -46,6 +47,7 @@
     private Path testFile5EntriesWithDuplicate;
     private JournalAbbreviationPreferences abbreviationPreferences;
     private DialogService dialogService;
+    private final JournalAbbreviationRepository repo = JournalAbbreviationLoader.loadBuiltInRepository();
```
Comment (tobiasdiez) [inline src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelNoShortestUniqueAbbreviationsTest.java:50]: rename here as well

File: src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelWithShortestUniqueAbbreviationsTest.java [hunk 5]
```diff
@@ -46,6 +47,7 @@
     private Path testFile5EntriesWithDuplicate;
     private JournalAbbreviationPreferences abbreviationPreferences;
     private DialogService dialogService;
+    private final JournalAbbreviationRepository repo = JournalAbbreviationLoader.loadBuiltInRepository();
```
Comment (tobiasdiez) [inline src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelWithShortestUniqueAbbreviationsTest.java:50]: dito

**Review by tobiasdiez — CHANGES_REQUESTED** [review]
Looks good to me, and I've only a few minor remarks 👿

File: src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java [hunk 6]
```diff
@@ -29,15 +30,15 @@ public JournalAbbreviationRepository(Path journalList) {
 
     private static boolean isMatched(String name, Abbreviation abbreviation) {
         return name.equalsIgnoreCase(abbreviation.getName())
-                || name.equalsIgnoreCase(abbreviation.getAbbreviation())
-                || name.equalsIgnoreCase(abbreviation.getMedlineAbbreviation())
-                || name.equalsIgnoreCase(abbreviation.getShortestUniqueAbbreviation());
+               || name.equalsIgnoreCase(abbreviation.getAbbreviation())
```
Comment (koppor) [inline src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java:33]: Reformat really necessary?

File: src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java [hunk 7]
```diff
@@ -29,15 +30,15 @@ public JournalAbbreviationRepository(Path journalList) {
...
+               || name.equalsIgnoreCase(abbreviation.getMedlineAbbreviation())
+               || name.equalsIgnoreCase(abbreviation.getShortestUniqueAbbreviation());
     }
 
     private static boolean isMatchedAbbreviated(String name, Abbreviation abbreviation) {
         boolean isAbbreviated = name.equalsIgnoreCase(abbreviation.getAbbreviation())
-                || name.equalsIgnoreCase(abbreviation.getMedlineAbbreviation())
-                || name.equalsIgnoreCase(abbreviation.getShortestUniqueAbbreviation());
+                                || name.equalsIgnoreCase(abbreviation.getMedlineAbbreviation())
```
Comment (koppor) [inline src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java:40]: Reformat really necessary?

File: src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java [hunk 8]
```diff
@@ -65,8 +66,8 @@ public boolean isAbbreviatedName(String journalName) {
         String journal = journalName.trim();
 
         return customAbbreviations.stream().anyMatch(abbreviation -> isMatchedAbbreviated(journal, abbreviation))
-                ||
-                abbreviationToFull.containsKey(journal);
+               ||
```
Comment (koppor) [inline src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java:69]: Reformat really necessary?

File: src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelMixedAbbreviationsTest.java [hunk 9]
```diff
@@ -197,11 +199,10 @@ void testBuiltInListsIncludeAllBuiltInAbbreviations() {
         viewModel.addBuiltInList();
         assertEquals(1, viewModel.journalFilesProperty().getSize());
         viewModel.currentFileProperty().set(viewModel.journalFilesProperty().get(0));
-        ObservableList<Abbreviation> expected = FXCollections
-                .observableArrayList(JournalAbbreviationLoader.getBuiltInAbbreviations());
+        ObservableList<Abbreviation> expected = FXCollections.observableArrayList(repo.getBuiltin());
         ObservableList<Abbreviation> actualAbbreviations = FXCollections
-                .observableArrayList(viewModel.abbreviationsProperty().stream()
-                                              .map(AbbreviationViewModel::getAbbreviationObject).collect(Collectors.toList()));
+                                                                        .observableArrayList(viewModel.abbreviationsProperty().stream()
```
Comment (koppor) [inline src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelMixedAbbreviationsTest.java:204]: Reformat really necessary?

File: src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelMixedAbbreviationsTest.java [hunk 10]
```diff
@@ -398,19 +399,19 @@ void testSaveAbbreviationsToFilesCreatesNewFilesWithWrittenAbbreviations() throw
 
         viewModel.saveJournalAbbreviationFiles();
         List<String> expected = Arrays.asList(
-                "Abbreviations;Abb;Abb",
-                "Test Entry;TE;T",
-                "MoreEntries;ME;M",
-                "JabRefTestEntry;JTE;JTE");
+                                              "Abbreviations;Abb;Abb",
```
Comment (koppor) [inline src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelMixedAbbreviationsTest.java:402]: Reformat really necessary?

File: src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelMixedAbbreviationsTest.java [hunk 11]
```diff
@@ -398,19 +399,19 @@ void testSaveAbbreviationsToFilesCreatesNewFilesWithWrittenAbbreviations() throw
...
         List<String> actual = Files.readAllLines(testFile4Entries, StandardCharsets.UTF_8);
 
         assertEquals(expected, actual);
 
         expected = Arrays.asList(
-                "EntryEntry;EE;EE",
-                "Abbreviations;Abb;Abb",
-                "Test Entry;TE;T",
-                "SomeOtherEntry;SOE;SOE");
+                                 "EntryEntry;EE;EE",
```
Comment (koppor) [inline src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelMixedAbbreviationsTest.java:411]: Reformat really necessary?

File: src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelNoShortestUniqueAbbreviationsTest.java (hunk 4 above)
Comment (koppor) [inline src/test/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModelNoShortestUniqueAbbreviationsTest.java:50]: Please rename to "repository"

File: src/test/java/org/jabref/logic/journals/AbbreviationsTest.java [hunk 12]
```diff
@@ -17,12 +17,12 @@ void setUp() {
     @Test
     void getNextAbbreviationAbbreviatesJournalTitle() {
         assertEquals("2D Mater.",
-                repository.getNextAbbreviation("2D Materials").get());
+                     repository.getNextAbbreviation("2D Materials").get());
```
Comment (koppor) [inline src/test/java/org/jabref/logic/journals/AbbreviationsTest.java:20]: Reformat really necessary?

File: src/test/java/org/jabref/logic/journals/AbbreviationsTest.java [hunk 13]
```diff
@@ -17,12 +17,12 @@ void setUp() {
...
     }
 
     @Test
     void getNextAbbreviationConvertsAbbreviationToDotLessAbbreviation() {
         assertEquals("2D Mater",
-                repository.getNextAbbreviation("2D Mater.").get());
+                     repository.getNextAbbreviation("2D Mater.").get());
```
Comment (koppor) [inline src/test/java/org/jabref/logic/journals/AbbreviationsTest.java:26]: Reformat really necessary?

**Review by koppor — CHANGES_REQUESTED** [review]
Mostly reormat comments - please check markdown

File: src/main/java/org/jabref/logic/journals/JournalAbbreviationLoader.java (hunk 2 above)
Comment (Siedlerchr) [reply src/main/java/org/jabref/logic/journals/JournalAbbreviationLoader.java:38]: This specific overload was never used locally. The

**Comment by Siedlerchr** [comment]
Thanks to calixtus the codeformatting is fine again. I also renamed the variables/method

**Pull request merged**